R2API/SoundAPI.cs

```diff
@@ -207,5 +216,86 @@ internal class Bank {
             /// </summary>
             internal static List<Bank> soundBanks = new List<Bank>();
         }
+
+        #endregion Soundbank Setup
+
+        #region NetworkSoundEventCatalog Setup
+
+        [R2APISubmoduleInit(Stage = InitStage.SetHooks)]
+        internal static void SetHooks() {
```

Comment:
Considering there's also a setHooks stage attribute on line 32, I'd like to see this renamed to something more descriptive to the networksoundeventcatalog. I suppose just `NetworkSetHooks` will suffice. Same goes for UnsetHooks.

Reply:
Didn't see commit comments carried through to the PR, my bad

Reply:
As noted, will be changing it.